moveit2_rolling.repos

```diff
@@ -1,13 +0,0 @@
-repositories:
-  geometric_shapes:
-    type: git
-    url: https://github.com/ros-planning/geometric_shapes
-    version: ros2
-  warehouse_ros:
-    type: git
-    url: https://github.com/ros-planning/warehouse_ros
-    version: ros2
-  warehouse_ros_mongo:
-    type: git
-    url: https://github.com/ros-planning/warehouse_ros_mongo
-    version: ros2
```

Comment:
I think we should be able remove `rolling.repos` I am not %100 sure CI would be happy, but if it is then it is good to remove this.

Reply:
see above. `rolling.repos` will be removed.